pkg/cache/cache.go

```diff
@@ -59,16 +60,18 @@ func (c *cache) InitCheck(checks v1.Canary) {
 	for _, check := range checks.Spec.GetAllChecks() {
 		key := checks.GetKey(check)
 		c.Checks[key] = pkg.Check{
-			Type:        check.GetType(),
-			Name:        checks.ID(),
-			Namespace:   checks.Namespace,
-			CanaryName:  checks.Name,
-			Description: check.GetDescription(),
-			Endpoint:    check.GetEndpoint(),
-			Interval:    checks.Spec.Interval,
-			Owner:       checks.Spec.Owner,
-			IconURL:     check.GetIconURL(),
-			Severity:    checks.Spec.Severity,
+			Type:         check.GetType(),
+			Name:         checks.ID(),
+			Namespace:    checks.Namespace,
+			Labels:       checks.ObjectMeta.Labels,
+			RunnerLabels: utils.GetLabels(),
```

Comment:
Utils generally are only for pure functions e.g. well defined input and output e.g. `utils.GetLabels("/etc/podinfo")` however due the fact that this is reading from disk we should do this at startup and cache the result e.g. 

`pkg/runner.go`
```golang
var Labels map[string]string = make(map[string]string

func LoadLabels(path string){}
```
and then in `cmd/operator.go`

```golang
runner.Labels = runner.LoadLabels("/etc/podinfo")
```